bin/civihr-install

```diff
@@ -596,8 +609,7 @@ export PATH="$PATH:$BINDIR"
 civihr_parse "$@"
 
 # Checking for Drush dependencies if not present downloading into temp.
-command -v drush >/dev/null 2>&1 || ( echo "[[Downloading dependencies.........]]" && curl -Lo "$TMPDIR/drush" "https://github.com/drush-ops/drush/releases/download/8.1.8/drush.phar"  && chmod +x "$TMPDIR/drush" && export PATH="$PATH:$TMPDIR/drush" )
-
+command -v drush >/dev/null 2>&1 || ( echo "[[Downloading dependencies.........]]" && curl -Lo "$TMPDIR/drush" "https://github.com/drush-ops/drush/releases/download/8.1.8/drush.phar" && chmod +x "$TMPDIR/drush" && export PATH="$TMPDIR:$PATH" )
```

Comment:
So, I forgot one important thing here: `export` will only affect the environment of the script itself and of its children. That is, any change you make will only have effect inside the script. So I think it's fine to call `export`.

The fix won't work though. The `export` is inside of a subshell (in between `( )`), meaning that the export will have effect only inside that subshell enviroment. My suggestion here is:

Create two new functions (it will also improve readbility):
```bash
function check_if_drush_is_available() {
  return command -v drush > /dev/null 2>&1
}

function install_drush() {
  echo "[[Downloading drush.........]]"
  curl -Lo "$TMPDIR/drush" "https://github.com/drush-ops/drush/releases/download/8.1.8/drush.phar"
  chmod +x "$TMPDIR/drush"
  export PATH="$TMPDIR:$PATH"
}
```

And then you can replace this line with:
```bash
check_if_drush_is_available || install_drush
```